tests/stdc/enc/enc.cpp

```diff
@@ -224,6 +223,33 @@ static void _test_time_functions(void)
 
         OE_TEST(after > before);
     }
+
+    /* Test nanosleep() with req==rem */
+    {
+        uint64_t before = oe_get_time();
+
+        /* Sleep for SLEEP_SECS seconds */
+        {
+            timespec ts = {SLEEP_SECS, 0};
+            OE_TEST(nanosleep(&ts, &ts) == 0);
+        }
+
+        uint64_t after = oe_get_time();
+
+        OE_TEST(after > before);
```

Comment:
Won't this always be true, regardless of whether the sleep worked or not?

Reply:
Agreed, although this is an existing problem with the test that was just replicated for new cases in this PR.

Reply:
In fact it occasionally fails without the fix. (oe_get_time returns milliseconds.)